File: Data Structures & Algorithms/remove-element/submission-0.go
func removeElement(nums []int, val int) int {
    var write int = 0 
    for read := 0; read < len(nums); read += 1 {
        if nums[read] != val {
            nums[write] = nums[read]
            write += 1
        }
    }
    return write
}

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array-ii/submission-0.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int write = 2;
        for(auto i = 2 ; i < nums.size(); ++i){
            if(nums[write-2] != nums[i]){
                nums[write] = nums[i];
                write++;
            }
        }
        return write;
    }
};